### PR TITLE
docs: FAQ Add a guide to allow all IP access

### PR DESCRIPTION
**Pull request by Firstsawyou**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

How to allow all IPs to access APISIX Dashboard is a common user question, and we should record it in the FAQ.
This PR is to record in the FAQ document how to allow all IPs to access APISIX Dashboard.

**Related issues**

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [x] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Review by iamayushdas — APPROVED** [review]
LGTM

**Comment by juzhiyuan** [comment]
@Firstsawyou Please update your PR's description template/

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -70,3 +70,29 @@ $ swagger generate spec -o ./docs/en/latest/api/api.yaml --scan-models
 ```shell
 $ swagger-markdown -i ./docs/en/latest/api/api.yaml
 ```
+
+### 6. How to allow all IPs to access APISIX Dashboard
+
+1. Allow all IPv4 access
+
+By default, the IPv4 range of `127.0.0.0/24` is allowed to access `APISIX Dashboard`. If you want to allow all IPv4 access, then just configure `conf.allow_list` in the configuration file of `conf/conf.yaml` as follows:
+
```
Comment (starsz) [inline docs/en/latest/FAQ.md:79]:
```
conf:
    allow_list:
```

can also work.

**Comment by Firstsawyou** [comment]
> @Firstsawyou Please update your PR's description template/

Updated.

File: docs/en/latest/FAQ.md (hunk 1 above)
Comment (Firstsawyou) [reply docs/en/latest/FAQ.md:79]: Updated.

**Review by starsz — APPROVED** [review]

File: docs/en/latest/FAQ.md [hunk 2]
```diff
@@ -70,3 +70,38 @@ $ swagger generate spec -o ./docs/en/latest/api/api.yaml --scan-models
...
+If you want to allow all IPs to access `APISIX Dashboard`, you only need to do the following configuration in the configuration file of `conf/conf.yaml`:
+
+```yaml
+conf:
+  allow_list:
+```
+
+Restart `manager-api`, all IPs can access `APISIX Dashboard`.
+
+Note: You can use this method in the exercise to allow all IPs to access your `APISIX Dashboard` instance, but it is not safe to use it in a production environment. In production, please only authorize specific IP addresses or address ranges to access your instance.
```
Comment (nic-chen) [inline docs/en/latest/FAQ.md:107]:
```suggestion
Note: You can use this method in development and test environment to allow all IPs to access your `APISIX Dashboard` instance, but it is not safe to use it in a production environment. In production, please only authorize specific IP addresses or address ranges to access your instance.
```

Comment (Firstsawyou) [reply docs/en/latest/FAQ.md]: Updated.

**Review by nic-chen — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Comment by imjoey** [comment]
The `Frontend e2e test` is always stuck, I filed a ticket to the Infra team to check if it has been set as `Required`. Put ticket https://issues.apache.org/jira/browse/INFRA-21906 here for tracking.

**Comment by LiteSun** [comment]
I just close and reopen this pr for retriggering the CI run.

**Comment by imjoey** [comment]
> I just close and reopen this pr for retriggering the CI run.

@LiteSun Thanks for doing this. To retrigger CI, we could also add an additional empty commit via `git commit --allow-empty -m "Retrigger CI"`.

**Comment by imjoey** [comment]
Apache Infra team has already fixed the problem which is always blocking CI. So merge this now.

**Pull request merged**